README.md

```diff
@@ -17,17 +17,29 @@ end
 
 ## Usage
 
+### Hover to reveal ActiveRecord schema
```

Comment:
These features are all listed as part of our documentation generated by RDoc. Should we link to there instead to avoid repeating ourselves?